src/vstart.sh

```diff
@@ -1162,7 +1162,22 @@ EOF
 	fi
         ceph_adm config-key set mgr/cephadm/ssh_identity_key -i ~/.ssh/id_rsa
         ceph_adm config-key set mgr/cephadm/ssh_identity_pub -i ~/.ssh/id_rsa.pub
-        ceph_adm mgr module enable cephadm
+        local retry_flag=0
+        while [ $retry_flag -lt 7 ]
```

Comment:
This condition is redundant: on the first iteration it's always true, when `retry_flag` gets up to 7, it's not checked because of `exit` in the loop body.
```suggestion
        while true; do
```

Reply:
Modified. Thanks for the advice.